nifi-nar-bundles/nifi-network-bundle/nifi-network-processors/src/main/resources/docs/org.apache.nifi.processors.network.ParseNetflowv5/additionalDetails.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
+<head>
+<meta charset="utf-8" />
+<title>Netflowv5Parser</title>
+<link rel="stylesheet" href="../../../../../css/component-usage.css"
+	type="text/css" />
+</head>
+
+<body>
+	<p>
+		Netflowv5Parser processor parses the ingress netflowv5 datagram format
+		and transfers it either as flowfile attributes or JSON object.
+		Netflowv5 format has predefined schema named "template" for parsing
+		the netflowv5 record. More information:&nbsp;<a title="RFC-netflowv5"
+			href="https://www.cisco.com/c/en/us/td/docs/net_mgmt/netflow_collection_engine/3-6/user/guide/format.html">RFC-netflowv5</a>
+	</p>
```

Comment:
Are you using "Template"  in the json you output?
I thought you removed that.  The schema should be what is in the record flow file exactly

Reply:
I removed "Template" relationship.. now, ONLY parsed data and "original" i output..  I updated this docs with "Template" json schema for user reference as per your review comments.

Reply:
The schema should match what is sent in the relationship.  Does the data that gets sent out ( the json you generate ) have the word "template" in it?

Reply:
no it doesn't.. Output json looks like this.
[data.txt](https://github.com/apache/nifi/files/2217192/data.txt)
So, you want to me change the docs with actual JSON schema(AVRO version)?? I thought of mentioning just the structure as given by RFC in json format hence i used word "template".

Reply:
So since this isn't a record processor/reader  I wouldn't put the avro schema there, we'll put the avro schema in that one.

I would put the exact json that you output, with values being the 'types' from your schema.
And above it, in the description just say that that is what you are doing.

Maybe follow that with an example:

" Here is the structure of the output net flow json, with the types:"
the json

" for example:"
the json with data

Reply:
Did you missed any attachments in your comments??  Are you referring something like this.. 
[dataschema.txt](https://github.com/apache/nifi/files/2217231/dataschema.txt)
Isn't it requires field level description??

Reply:
yes that is the type of thing I'm referring to.
If you want to put the description, put it outside maybe.

Reply:
@bbende do you have an opinion on how to document the json?

Reply:
I think something like what is in dataschema.txt is fine. If we think people are going to use the record processors on the output, then it wouldn't hurt to also have the Avro schema, but not totally necessary.

Reply:
I'll put the avro in the record reader's additionalDetails.  I think it is confusing to have it with the processor.

Reply:
any other changes required??